worker/src/RTC/PipeTransport.cpp

```diff
@@ -593,7 +594,7 @@ namespace RTC
 
 		if (HasSrtp() && !this->srtpRecvSession->DecryptSrtp(const_cast<uint8_t*>(data), &intLen))
 		{
-			RTC::RtpPacket* packet = RTC::RtpPacket::Parse(data, static_cast<size_t>(intLen));
+			auto packet = RTC::RtpPacket::Parse(data, static_cast<size_t>(intLen));
```

Comment:
Let's please use `auto*` for pointer for readability and consistency.

Reply:
From the point of view of casual reader both auto/auto* is not really clear here. What we have - function parse which returns something: guesses taking into account class name includes RTPPacket, RTPPacket*, RTPPacket::SharedPtr or RTPPacket::SharedPtr* or std::unique/shared_prr. So, maybe it would be better in cases where it's not 100% obvious what type is  to write explicit type?